### PR TITLE
refactor: improve notebook login ui

### PR DESCRIPTION
**Pull request by Andrei997**
Ref #78 
This PR is to improve the UX of the `notebook_login()` feature.

### CODE REVIEW

**Comment by Andrei997** [comment]
This is how the new UI looks like.

Screen Shot 2022-10-14 at 18 50 45

**Comment by Andrei997** [comment]
Regarding coverage, we cannot rely test the `notebook_login()` function, since it's only meant to run in notebook environments. The main `long_sync()` is tested though.

**Comment by bwanglzu** [comment]
looks good, tested in Finetuner client:
EBB0A2CC-FE0C-4381-A77A-74C62B918D82

**Comment by Andrei997** [comment]
Ready to be merged, after we merge will update the bump the SDK version in Jina core.

**Review by delgermurun — APPROVED** [review]
🎉 Nice!

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: hubble/utils/auth.py [hunk 1]
```diff
@@ -17,64 +17,128 @@
...
+        }}
+        .spaced {{
+            margin: 20px 0;
+        }}
+    </style>
+    <center>
+        <img src={JINA_LOGO} width=175 alt='Jina AI'>
+        <div class='spaced'></div>
+        <p>
+            Copy a <b>Personal Access Token</b>, paste it below, and press the <b>Login with Token</b> button.
```
Comment (alexcg1) [inline hubble/utils/auth.py:51]: This should be "Log in". And what case are we using on buttons elsewhere? Should we use capital T on Token?

File: hubble/utils/auth.py [hunk 2]
```diff
@@ -17,64 +17,128 @@
...
+        }}
+        .spaced {{
+            margin: 20px 0;
+        }}
+    </style>
+    <center>
+        <img src={LOGO} width=175 alt="Jina AI">
+        <div class='spaced'></div>
+        <p>
+            Please open the following <a href='{HREF}' target='_blank'>link</a> to continue the login process.
```
Comment (alexcg1) [inline hubble/utils/auth.py:138]:
```suggestion
            Please open <a href='{HREF}' target='_blank'>this link</a> to continue the login process.
```

**Review by alexcg1 — CHANGES_REQUESTED** [review]

File: hubble/utils/auth.py (hunk 1 above)
Comment (Andrei997) [reply hubble/utils/auth.py:51]: From what I've seen in other places we use **Login**, and I also think it should be **Login**. 
It this context it can be seen as a noun phrase, so it's not necessarily wrong. Plus, I think **Log in with token** is too wordy for a button. 
If you want, we can switch to "Token login" and "Browser login".

Screen Shot 2022-10-14 at 21 09 11

Comment (alexcg1) [reply hubble/utils/auth.py:51]: > If you want, we can switch to Token login and Browser login.

I like this

**Comment by Andrei997** [comment]
Updated UI 
Screen Shot 2022-10-14 at 22 05 43

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:
